README.md

```diff
@@ -12,7 +12,7 @@ Installation
 ------------
 
 Standard DITA-OT plug-in installation,
-see [DITA-OT documentation](http://www.dita-ot.org/2.5/dev_ref/plugins-installing.html).
+see [DITA-OT documentation](https://www.dita-ot.org/3.6/topics/plugins-installing.html).
```

Comment:
Let's point to the latest version of the docs here, rather than pinning to a particular version:

```suggestion
see [DITA-OT documentation](https://www.dita-ot.org/dev/topics/plugins-installing.html).
```